gradle.properties

```diff
@@ -0,0 +1,4 @@
+org.gradle.jvmargs=-Xmx6g -XX:MaxPermSize=6g -XX:ReservedCodeCacheSize=2g -Dfile.encoding=UTF-8
```

Comment:
this needs, at the very least, some comments explaining why we need to set these properties. Are they for CI purposes?

Reply:
This is a pretty standard configuration for multi-module projects—new Android Studio projects already include some of this, though I've added properties to improve multi-module perf. You can read more about the properties here: https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_configuration_properties.

Reply:
MaxPermSize doesn't exist anymore on java 8+ and doesn't have any effect

Reply:
Good point, thanks!